src/routes/api/room.ts

```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import { createRoom, queryRoom } from '../../controllers/room';
```

Comment:
My current thoughts are that refactoring to controllers is a bit pointless here since we're already using Routers to split up our endpoints. I'm curious what others think about this.